src/clr-angular/modal/modal.ts

```diff
@@ -9,22 +9,18 @@ import {
   EventEmitter,
   HostBinding,
   HostListener,
+  Inject,
   Input,
   OnChanges,
   OnDestroy,
   Output,
   SimpleChange,
   ViewChild,
-  Inject,
-  ElementRef,
-  PLATFORM_ID,
 } from '@angular/core';
-
 import { FocusTrapDirective } from '../utils/focus-trap/focus-trap.directive';
-import { ScrollingService } from '../utils/scrolling/scrolling-service';
 import { ClrCommonStringsService } from '../utils/i18n/common-strings.service';
 import { UNIQUE_ID, UNIQUE_ID_PROVIDER } from '../utils/id-generator/id-generator.service';
-import { isPlatformBrowser } from '@angular/common';
+import { ScrollingService } from '../utils/scrolling/scrolling-service';
```

Comment:
Unused?

Reply:
This is still in use. It's just the line position that's been changed.